### ISSUE TITLE
Add arrow keys to view mode

### ISSUE THREAD

**Issue by EpocSquadron** [helix-editor/helix#977]
In normal mode `hjkl` are equivalent to the respective arrow keys for navigating around. In view mode, only `hjkl` are recognized. We should add them for consistency.

**Comment by matu3ba** [comment]
I would argue against adding anti-patterns that users later must remove and unlearn.

**Comment by EpocSquadron** [comment]
> I would argue against adding anti-patterns that users later must remove and unlearn.

We get to decide what the good patterns are as is a new editor. Besides, consider my use case: I have a split keyboard that's programmable with QMK. I use an alternative layout for the alpha keys (dvorak is the common case though I use a niche one noone had heard of) so hjkl are not under the right hand like they are on qwerty. To get around this I have a layer key (like function key on a laptop) which transforms the keys in the places you expect to find hjkl with the corresponding arrow keys. Thus, I can use my hjkl muscle memory on any context, including non-vim contexts. Because this is more applicable and the number of layers I need to fit all the keys I want is limited (both by number of keys and mental effort to juggle them), it doesn't make sense for me to make another layer just to put hjkl on those places, which would then mean pressing a different layer key depending on my context to get the desired behavior. 

So in other words, I'm not using the anti pattern you're arguing against at all, in fact I am using the same ergonomic movements on the home row to move up and down, but they happen to show to the operating system as arrow keys.

I know this is a common use case for users of non-qwerty layouts. This is also an accessibility thing because non-qwerty can also mean "German" or "Turkish".

**Comment by python128** [comment]
Opened a PR, please have a look.

**Comment by matu3ba** [comment]
> So in other words, I'm not using the anti pattern you're arguing against at all, in fact I am using the same ergonomic movements on the home row to move up and down, but they happen to show to the operating system as arrow keys.

LGTM then and sounds like a good decision. I was not aware of this use case, so my assumption that it is an antipattern was too fast judging.
